CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.1.0] - TBD
+### Added
+- Added `spotbugs-maven-plugin`, version 3.1.12.2.
+### Changed
+- Updated `eg-oss-plugin-config` version to 1.1.0 (was 1.0.0).
+
 ## [1.0.1] - TBD
```

Comment:
Because this 1.0.1 release is TBD and was never actually done, it can be merged with the 1.1.0 release, so you can just remove this line.